internal/controller/standalone_pgadmin/pod.go

```diff
@@ -342,3 +343,31 @@ if os.path.isfile('` + ldapPasswordAbsolutePath + `'):
 
 	return append([]string{"bash", "-ceu", "--", script, "startup"}, args...)
 }
+
+// PodSecurityContext returns a v1.PodSecurityContext for pgadmin that can write
+// to PersistentVolumes.
+func PodSecurityContext(r *PGAdminReconciler, cluster *v1beta1.PGAdmin) *corev1.PodSecurityContext {
+	podSecurityContext := initialize.PodSecurityContext()
+
+	// // Use the specified supplementary groups except for root. The CRD has
```

Comment:
Do we want to take this commented out code out? Put a link to a ticket to revisit this issue as necessary? 💭